oleesir/banking-app
# ft(credit-debit-page):implement credit-debit-transaction-template

#### What does this PR do?
implement credit-debit-transaction-template
#### Description of Task to be completed?
implement credit-debit-transaction-template
#### How should this be manually tested?
* Navigate to the UI folder and click on bank_accounts.html in your browser
* You will see an eye icon click eye icon
#### What are the relevant pivotal tracker stories?
[#164728667](https://www.pivotaltracker.com/story/show/164728667)
#### Any background context you want to add?
N/A
#### Screenshots
<img width="1280" alt="Screenshot 2019-03-29 at 12 46 17 PM" src="https://user-images.githubusercontent.com/17679511/55230980-afe61280-5221-11e9-8afa-8a0e09b01f2e.png">
